config/source_input_services_test.go

```diff
@@ -0,0 +1,233 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
```suggestion
	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"
```

Reply:
I really don't know why my goimports keeps doing this :S, it's pretty annoying